lib/human.rb

```diff
@@ -0,0 +1,22 @@
+module TicTacToe
+  class Human < Player
+
+    def make_move(board)
+      @console_ui.move_messages(board.cells, @marker)
+      @console_ui.move_selection_message
+      move = @console_ui.user_input
+      move = valid_move_loop(move, board)
+      board.fill(move, @marker)
```

Comment:
I believe `ComputerPlayer`'s `#make_move` is returning the Board object. The return type for any type of Player that implements `#make_move` should be consistent so that any caller of a Player object can safely use it without having to know what type of player it is. We can talk about this more in person too, if you'd like